design/release-notes/release-0.9/draft-release-notes.md

```diff
@@ -1,5 +1,46 @@
+The v0.9 release is one of our biggest yet, packed with new features and bug
+fixes!
+
+The introduction of the new CertificateRequest resource type is significant as
+it is a step towards where we want to be for 1.0, defining an API specification
+for Certificates and allowing anyone to implement their own issuers and CAs as
+first class citizens.
+
+This release includes changes from:
+
+* Aaron Gershman
+* Aled James
+* Artem Yarmoluk
+* Carlos Panato
+* Chris Abiad
+* Christopher Abiad
+* Crystal-Chun
+* Dan
+* Dobes Vandermeer
+* Hans Kristian Flaatten
+* Hays Clark
+* Ivan Wallis
+* James Munnelly
+* JoshVanL
```

Comment:
My full name would be nice :wink: 
`Joshua Van Leeuwen`

Reply:
👍 these were pulled using ```git log --pretty=format:"%an" v0.8.0..master | sort | uniq```

Reply:
Guess I have something wrong set somewhere then